README.md

```diff
@@ -8,10 +8,43 @@ Poky GENIVI compliant.
 Branch Policy
 -------------
 
-Please note:
-- The most recent development is done on the highest numbered branch
-- Master branch is used as a stable version, typically equal to the most recently *released* numbered branch
-- Maintenance patches are added to both master and released branches
+- New development is done on **master** branch and most new change requests
+(PRs) should be proposed as changes to master, unless you know they are
+applicable to a particular release only.
+- Somewhere near a new major release, a numbered release branch (e.g. 14.0)
+is created from master, and the new branch goes into stabilization/release phase.
+- Release numbers use [semantic versioning](http://semver.org/), and are
+potentially also tagged with GENIVI-specific release tags, such as "M-0.2"
+for the second baseline prerelease for the "M" platform release.
+- After a versioned release, the release branch remains for maintenance and
+updates.
+- The project maintainer shall ensure that relevant patches are
+cherry-picked to every branch where they apply. I.e. patches should be
+back-ported *at minimum* to the Support Window versions as defined below.
+- In general, prefer a linear commit history (rebase and cherry-pick),
+  applying merge commits only where absolutely necessary to sort out a
+  complex merge situation (which we should rarely have).
+
+Support Window (Bugfixes, improvements)
+---------------------------------------
+
+- Because of available resources, and often low engagement from those companies
+that have settled on a version and gone into a production project, support
+is given for only the most recently released version, and *one* version
+before it.
+
+Support Window (Security)
+-------------------------
+
+- Critical security issues should be fixed in *two* versions behind the most
+recently released.
```

Comment:
Hopefully that will not be read as that critical security issues are being fixed two versions behind _for_ people. That some how it is falling on the maintainer/BIT shoulders to find/monitor security fixes and apply them.

I think that is what you try to address in the following paragraph. Making it more of a statement of how far back the project would normally be taking in fixes from the community based on current resources.

That said, I'm not sure how I might rewrite this section apart from possible inverting the text in some way. Maybe the problem is the "should" in the first para and what is could be interpreted as.